lib/Enums/ObjectTypeIDs.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace HubSpot\Enums;
+
+class ObjectTypeIDs
+{
+    public const COMPANIES = '0-2';
+    public const CONTACTS = '0-1';
+    public const DEALS = '0-3';
+    public const TICKETS = '0-5';
+    public const APPOINTMENTS = '0-421';
+    public const CALLS = '0-48';
+    public const COMMUNICATIONS = '0-18';
+    public const COURSES = '0-410';
+    public const EMAILS = '0-49';
+    public const FEEDBACK_SUBMISSIONS = '0-19';
+    public const INVOICES = '0-53';
+    public const LEADS = '0-136';
+    public const LINE_ITEMS = '0-8';
+    public const LISTINGS = '0-420';
+    public const MARKETING_EVENTS = '0-54';
+    public const MEETINGS = '0-47';
+    public const NOTES = '0-46';
+    public const ORDERS = '0-123';
+    public const PAYMENTS = '0-101';
+    public const POSTAL_MAIL = '0-116	';
```

Comment:
Isn't the tabulator overhead?

```suggestion
    public const POSTAL_MAIL = '0-116';
```